src/common/io.ts

```diff
@@ -317,16 +318,12 @@ export class IOReadable implements AoIORead {
   async getGatewayDelegateAllowList({
     address,
     ...pageParams
-  }: {
-    address: WalletAddress;
-  } & PaginationParams<WalletAddress>): Promise<
-    PaginationResult<WalletAddress>
-  > {
+  }: AoPaginatedAddressParams): Promise<PaginationResult<WalletAddress>> {
     return this.process.read<PaginationResult<WalletAddress>>({
       tags: [
         { name: 'Action', value: 'Paginated-Allowed-Delegates' },
         { name: 'Address', value: address },
-        ...paginationParamsToTags<WalletAddress>(pageParams),
+        ...paginationParamsToTags(pageParams),
```

Comment:
these methods were duplicated, but the type signatures were slightly different. the pagination type helpers are a bit hard to use